ddl/dist_backfilling.go

```diff
@@ -205,9 +205,6 @@ func runBackfillJobs(d *ddl, sess *session, ingestBackendCtx *ingest.BackendCont
 	}
 	workerCnt = len(workerCtx.backfillWorkers)
 	bwMgr := newBackfilWorkerManager(workerCtx)
-	d.backfillWorkerPool.SetConsumerFunc(func(task *reorgBackfillTask, _ int, bfWorker *backfillWorker) *backfillResult {
```

Comment:
I need this to pass a parameter to fix a bug. In addition, since there is a second issue, don't you want to change this for now?